strapi/strapi
# enhancement: allow relative links in the Blocks editor

### What does it do?

Allows authors to create relative links in the blocks editor

### Why is it needed?

Currently, links cannot be saved when entering relative links in the blocks editor.

### How to test it?

- open the new blocks editor
- create a link
- enter any relative link as the link target (e.g. /about-strapi)
expected: the user can save the link and save the block

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/18562
